docs/tiling.md

```diff
@@ -0,0 +1,337 @@
+# Integration of TileJSON with MicroJSON
+## Purpose
+This specification outlines how to use TileJSON to integrate tiled MicroJSON data. It provides examples of how TileJSON can be used to specify the tiling scheme and zoom levels for MicroJSON data. It is based on the [TileJSON 3.0.0 specification](https://github.com/mapbox/tilejson-spec/blob/master/3.0.0/README.md), but extends it by recommending additional properties include the integration of MicroJSON data and fit purposes of microscopy data visualization. Recommendations made here are NOT part of the original TileJSON specification, but are made to fit the purposes of microscopy data visualization and integration with MicroJSON, although all recommendations are made to be compatible with the original TileJSON specification.
+
+## Background of TileJSON
+
+[TileJSON](https://github.com/mapbox/tilejson-spec/) is a widely-used format in mapping applications for specifying tilesets. Developed to streamline the integration of different map layers, TileJSON is essential for ensuring consistency across mapping platforms. It describes tilesets through a JSON object, detailing properties like tile URLs, zoom levels, and spatial coverage.
+
+
+## TileJSON for MicroJSON Object Structure
+
+- **`tilejson`:** Specifies the version of the TileJSON spec being used. Required for all TileJSON objects.
+- **`name`:** The name of the tileset. Optional but recommended.
+- **`description`:** Provide a brief description of the tileset. Optional but recommended.
+- **`version`:** The version of the tileset. Optional but recommended.
+- **`attribution`:** A link to the data source or other attribution information, e.g. organisational origin. Optional but recommended.
+- **`tiles`:** Required. The URL pattern for accessing the vector tiles. The  `urlbase/{zlvl}/{t}/{c}/{z}/{x}/{y}` is the recommended default naming pattern for the tiles, in this order, where `urlbase` is the base URL (e.g. `http://example.com/tiles`), `{zlvl}` is the zoom level, `{t}` is the tileset timestamp, `{c}` is the channel, `{z}` is the z coordinate, and `{x}` and `{y}` are the x and y coordinates, respectively. 
+- **`minzoom` and `maxzoom`:** Defines the range of zoom levels for which the tiles are available. 
+- **`bounds`:** Optional. Specifies the geomoetrical bounds covered by the tileset. Specified as an array of four numbers in the order `[minX, minY, maxX, maxY]`.
```

Comment:
replace covered by
included in the dataset

Reply:
Changed